### ISSUE TITLE
Resetting ignored words

### ISSUE THREAD

**Issue by cucumberry** [ropensci/hunspell#30]
If I ignore a word with the `ignore` argument, it remains ignored in future checks:

```r
> hunspell("ggggggg")
[[1]]
[1] "ggggggg"

> hunspell("ggggggg", ignore = "ggggggg")
[[1]]
character(0)

> hunspell("ggggggg")
[[1]]
character(0)
```

But if I ignore it with `dict = dictionary(add_words = )`,  it doesn't stay ignored:

```r
> hunspell("fffffff")
[[1]]
[1] "fffffff"

> hunspell("fffffff", dict = dictionary(add_words = "fffffff"))
[[1]]
character(0)

> hunspell("fffffff")
[[1]]
[1] "fffffff"
```

I don't know which of the two behaviours is preferable, but they should probably be consistent, and it would be good to be able to reset the words added with `ignore`.

**Comment by jeroen** [comment]
Ah that's a caching bug. Thanks for reporting.

**Comment by jeroen** [comment]
Fixed again in 2.9.
